test/gpu/native/distArray/blockInsideOn.chpl

```diff
@@ -1,6 +1,14 @@
 use BlockDist;
 use GPUDiagnostics;
 
+proc verifyLaunches() {
```

Comment:
It's kind of unfortunate that we have this function copied and pasted (sans the values for expected) across several different tests.

I'm not sure I like the idea of factoring it out into its own "GpuTestingUtil" module or something better or not though.

Reply:
Yeah, I never know what to do with stuff like this (use "`assertOnGpu()`!!" is the obvious answer).

If we move it to a module in `test/gpu/native`, then it is a burden, too. I get annoyed when I can't compile a test without proper `-M` flag. (streamPrototype/stream is like that today). But copy/paste is never a good idea either. Our GPU tests have always had some clunkiness, and I am willing to tolerate this for today. But I can be convinced otherwise.

Reply:
Yeah my thoughts are the same as yours here, I really want to minimize the number of tests where we need special `.compopts`/`COMPOPTS` (I know about `start_tests` but sometimes I like to be able to repro a failing test by running the `chpl` command explicitly and the simpler it is the better).

We don't have to pass `-M` if the module file is in the same dir right? So we could add a `test/gpu/native/gpuTestTools.chpl` module and add symbolic links to it where we need it. But again, uggh, that's repulsive in its own way.

Anyway, I think I'd leave it as you have it in this PR.